rsocket/internal/Common.cpp

```diff
@@ -107,15 +131,20 @@ void ResumeIdentificationToken::set(std::vector<uint8_t> newBits) {
   bits_ = std::move(newBits);
 }
 
-std::ostream& operator<<(
-    std::ostream& out,
-    const ResumeIdentificationToken& token) {
+std::string ResumeIdentificationToken::str() const {
+  std::stringstream out;
   out << "0x";
-  for (auto b : token.data()) {
+  for (auto b : bits_) {
     out << HEX_CHARS[(b & 0xF0) >> 4];
     out << HEX_CHARS[b & 0x0F];
   }
-  return out;
+  return out.str();
+}
+
+std::ostream& operator<<(
+    std::ostream& out,
+    const ResumeIdentificationToken& token) {
+  return out << token.str();
```

Comment:
I think it should be the other way.  We shouldn't need to create a string to write to an `ostream`, especially if the `.str()` method is working with an `ostream` internally.